tools/msp_sr_readmap_and_size_histograms/test-data/Readmap_dataframe.tab

```diff
@@ -2373,13 +2547,21 @@ FBti0020401	5811	17.0	F	sample3.srbowtie_out
 FBti0020401	5996	-1.0	R	sample3.srbowtie_out
 FBti0020401	6183	-1.0	R	sample3.srbowtie_out
 FBti0020401	6218	1.0	F	sample3.srbowtie_out
+FBti0020401	6348	0	F	sample3.srbowtie_out
+FBti0020406	0	0	F	sample3.srbowtie_out
 FBti0020406	161	1.0	F	sample3.srbowtie_out
 FBti0020406	697	-1.0	R	sample3.srbowtie_out
+FBti0020406	812	0	F	sample3.srbowtie_out
+FBti0019511	0	0	F	sample3.srbowtie_out
```

Comment:
Ah, yes, it is correct after all:
https://github.com/ARTbio/tools-artbio/pull/79/files#diff-2a5423b1092f0853a89d34f45bc3ee65R1335